feature-utils/poly-analysis/src/analysis/report/report-analysis.js

```diff
@@ -0,0 +1,14 @@
+import RootAnalysis from "../root-analysis.js";
+
+export default class ReportAnalysis extends RootAnalysis {
+    get isForDataReport() {
```

Comment:
This is essentially the only functionality this adds, right? Couldn't simply be an attribute?

Reply:
I'd prefer refactoring this code more substantially after moving it - it's already very hard to keep up with the _actual_ changes, with all the moving around.